driver-core/src/main/com/mongodb/operation/ParallelCollectionScanOperation.java

```diff
@@ -187,7 +223,11 @@ private BsonDocument getCursorDocument(final BsonDocument cursorDocument) {
     }
 
     private BsonDocument getCommand() {
-        return new BsonDocument("parallelCollectionScan", new BsonString(namespace.getCollectionName()))
+        BsonDocument document = new BsonDocument("parallelCollectionScan", new BsonString(namespace.getCollectionName()))
                .append("numCursors", new BsonInt32(getNumCursors()));
+        if (readConcern.getValue() != null) {
```

Comment:
The readConcern property already defaults to ReadConcern.DEFAULT, so in practice how often will this be null?  I think it would be more appropriate to force readConcern to be non-null, and instead check here whether readConcern.isServerDefault()